src/build-scripts/build_OpenJPEG.bash

```diff
@@ -32,16 +32,22 @@ if [[ ! -e ${OPENJPEG_SRC_DIR} ]] ; then
     git clone ${OPENJPEG_REPO} ${OPENJPEG_SRC_DIR}
 fi
 cd ${OPENJPEG_SRC_DIR}
-echo "git checkout ${OPENJPEG_VERSION} --force"
-git checkout ${OPENJPEG_VERSION} --force
+
+if [[ -z $OIIO_DEP_BUILD_ONLY ]]; then
+    echo "git checkout ${OPENJPEG_VERSION} --force"
+    git checkout ${OPENJPEG_VERSION} --force
+fi
```

Comment:
I think you may have missed a bit here. You're guarding the "git checkout", which only syncs which commit is active in the already-local repo. It's the "git clone" several lines above that does the I/O.

But is OIIO_DEP_BUILD_ONLY really necessary? The clone only happens if the directory isn't already there. Like, if you run the existing script twice, I believe that the second time it wouldn't download anything.